profile/README.md

```diff
@@ -1 +1,28 @@
-# Ziggurat
+<p align="center">
+    <img src="assets/logo.png" height="300px">
+</p>
+<br />
+Ziggurat is a collection of network testing tools and a suite of conformance, performance and resistance tests our team developed in Rust.
+
+# About
+
+The project is developed by [Equilibrium](https://equilibrium.co) that provides [zcashd](https://github.com/zcash/zcash) and [zebra](https://github.com/ZcashFoundation/zebra) developers with the foundation of reliability, performance, and scalability needed to realize the true potential of the Zcash network.
```

Comment:
We should make this generic—we're not only testing Zcash these days 😉

Reply:
The full list is ZCash, Algorand, and XRPL Ledger. Hopefully Polkadot soon too 🤞